gammapy/modeling/models/tests/test_io.py

```diff
@@ -483,3 +484,19 @@ def test_compound_models_io(tmp_path):
         "model.spectral.alpha",
         "model.spectral.beta",
     ]
+
+
+def test_meta_io(caplog, tmp_path):
+    m = PowerLawSpectralModel()
+    sk = SkyModel(spectral_model=m, name="model")
+    Models([sk]).write(tmp_path / "test.yaml")
+
+    sk_dict = read_yaml(tmp_path / "test.yaml")
+    assert "metadata" in sk_dict
+
+    sk_dict["metadata"]["creator"] = "Gammapy 1.1"
```

Comment:
Here, if you do not define it as "Gammapy 1.1" it would be set by default as dev version by the `from_default` metadata creator? Perhaps you can also test that the warning is present in that case in `test.yaml`

Reply:
done